sorts/recursive_selection_sort.py

```diff
@@ -0,0 +1,26 @@
+def minind(a, i, j):
+    if i == j:
+        return i
+    k = minind(a, i + 1, j)
+    return i if a[i] < a[k] else k
+
+
+def RecSelectionSort(a, n, index=0):
```

Comment:
```suggestion
def recursive_selection_sort(a, n, index=0):
```
Python variable names use snake_case